### PR TITLE
Teleop: adding modem ranking info

### PR DESCRIPTION
**Pull request by hugomatic**

### CODE REVIEW

File: modules/dreamview/backend/teleop/teleop.cc [hunk 1]
```diff
@@ -119,6 +123,11 @@ void TeleopService::Start() {
       [this](const std::shared_ptr<DaemonServiceRpt> &msg) {
         UpdateOperatorDaemonRpt(msg);
       });
+
+  pad_message_writer_ =
+      node_->CreateWriter<PadMessage>(planning_pad_channel);
+
+
```
Comment (xiaoxq) [inline modules/dreamview/backend/teleop/teleop.cc]: Please run `./scripts/clang-format.sh` to fix style issues.

File: modules/dreamview/backend/teleop/teleop.cc [hunk 2]
```diff
@@ -40,7 +43,8 @@ const std::string modem0_id = "0";
 const std::string modem1_id = "1";
 const std::string modem2_id = "2";
 
-const unsigned int encoder_count = 2;
+const unsigned int encoder_count = 3;
+const unsigned int write_wait_ms = 50;
```
Comment (xiaoxq) [inline modules/dreamview/backend/teleop/teleop.cc]: Prefer `static`, and names like `kEncoderCount`.

File: modules/dreamview/backend/teleop/teleop.cc (hunk 1 above)
Comment (hugomatic) [reply modules/dreamview/backend/teleop/teleop.cc]: fixed [here](https://github.com/ApolloAuto/apollo/pull/10228/commits/f45083489676b39d68f3d82468216844c30de7d1)

File: modules/dreamview/backend/teleop/teleop.cc (hunk 2 above)
Comment (hugomatic) [reply modules/dreamview/backend/teleop/teleop.cc]: fixed [here](https://github.com/ApolloAuto/apollo/pull/10228/commits/f45083489676b39d68f3d82468216844c30de7d1)

**Pull request merged**